x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/generative/GenerativeRestTest.java

```diff
@@ -58,6 +58,7 @@ public abstract class GenerativeRestTest extends ESRestTestCase implements Query
         "Unbounded SORT not supported yet",
         "The field names are too complex to process", // field_caps problem
         "must be \\[any type except counter types\\]", // TODO refine the generation of count()
+        "INLINE STATS cannot be used after LIMIT command",
```

Comment:
I'm good with this 👍 

Maybe we could be a bit smarter and just skip INLINE STATS generation if there is an explicit LIMIT or a FORK, eg.

```
public class InlineStatsGenerator

...

    @Override
    public CommandDescription generate(List<CommandDescription> previousCommands, List<Column> previousOutput, QuerySchema schema, QueryExecutor executor) {
        if (previousCommands.stream()
            .anyMatch(c -> c.commandName().equals(LimitGenerator.LIMIT) || c.commandName().equals(ForkGenerator.FORK))) {
            return EMPTY_DESCRIPTION;
        }
        return super.generate(previousCommands, previousOutput, schema, executor);
    }
```

But I'm not sure it will catch all the cases, maybe other generators introduce a limit. 
Maybe we should just have both.